packages/test-utils/lib/test-utils.js

```diff
@@ -8,20 +8,29 @@ const { MongooseAdapter } = require('@keystone-alpha/adapter-mongoose');
 
 const SCHEMA_NAME = 'testing';
 
-function setupServer({ name, adapterName, createLists = () => {} }) {
+function setupServer({ name, adapterName, createLists = () => {}, createApps, keystoneOptions }) {
   const Adapter = { mongoose: MongooseAdapter, knex: KnexAdapter }[adapterName];
-  const args = { mongoose: {}, knex: { dropDatabase: true } }[adapterName];
+  const args = {
+    mongoose: {},
+    knex: {
+      dropDatabase: true,
+      knexOptions: { connection: process.env.KNEX_URI || 'postgres://localhost/keystone' },
```

Comment:
These are the defaults anyway, so we don't need to set them here.

```suggestion
      knexOptions: {},
```

Reply:
Nope, we need this makes the database for all tests `keystone` rather than the test name. We could make it `test` or something else, but this would require everyone to make a postgres db named `test` for testing.

Reply:
Ahh, gotchya 👍